src/js/twitter.js

```diff
@@ -8,5 +10,47 @@
   js = d.createElement(s)
   js.id = id
   js.src = p + '://platform.twitter.com/widgets.js'
+  js.setAttribute('onload', 'twitter_filter(' + id + ')')
```

Comment:
Using `twitter_filter(id)` should be possible right?

Reply:
js.id is defined as id a few lines above so they should be both equal to the same thing - i chosen id since it was shorter

Reply:
Yeah, I mean referencing the function directly, without using string.

Reply:
My apologies, i understand now, i’ll give it a go now